spec/policies/request_link_policy_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+RSpec.describe RequestLinkPolicy do
+  subject(:policy) { described_class.new(location:, library:, items:) }
+
+  let(:location) { 'HOOVER' }
+  let(:library) { 'OVERSIZED' }
+
+  describe '#aeon_pageable?' do
+    subject { policy.aeon_pageable? }
+
+    context 'when there are no items' do
+      let(:items) { [] }
+
+      it { is_expected.to be true }
+    end
+  end
```

Comment:
out of context, this isn't intuitive to me: something is aeon pageable if there are no items?

Reply:
I agree, but I think that was the previous behavior.

Reply:
It seems that @cbeer has confirmed this was a data problem and has worked to resolve it.